src/components/Header/index.tsx

```diff
@@ -79,3 +59,46 @@ export function Header() {
         </header>
     );
 }
+
+export function PrimaryNav({ colorMode, toggleColorMode }: Props) {
+    return (
+        <div className="d-flex justify-content-between w-100">
+            <Navbar.Brand as={Link} href="/">
+                {SITE_NAME}
+            </Navbar.Brand>
+            <Navbar.Toggle aria-controls="main-navbar-nav" />
+            <Nav className="d-none d-lg-flex align-items-center">
+                <ThemeToggleButton
+                    colorMode={colorMode}
+                    toggleColorMode={toggleColorMode}
+                />
+            </Nav>
+        </div>
+    );
+}
+
+export function MobileNav({ colorMode, toggleColorMode }: Props) {
+    return (
+        <Navbar.Offcanvas id="main-navbar-nav" placement="end">
+            <Offcanvas.Header closeButton>
+                <Offcanvas.Title className="d-flex align-items-start">
+                    <ThemeToggleButton
+                        colorMode={colorMode}
+                        toggleColorMode={toggleColorMode}
+                        mobileNav
+                    />
+                </Offcanvas.Title>
+            </Offcanvas.Header>
+            <Offcanvas.Body>
+                <span className="d-lg-none d-lg-none d-xl-none fs-4 fw-bold">
```

Comment:
There are two `d-lg-none`. `lg` addition means apply for large screens and up. So there is no need for `d-xl-none`

Reply:
done